### PR TITLE
Add new widget in the VZE dashboard

### PR DESCRIPTION
**Pull request by patrickm02L**
## Associated issues
This PR closes [#11880](https://github.com/cityofaustin/atd-data-tech/issues/11800). 

Staff wanted a link on the dashboard for quick access.

## Testing
**URL to test:** [Netlify](https://deploy-preview-1201--atd-vze-staging.netlify.app/#/dashboard)

**Steps to test:**

Click on the new "Access Management Crashes" widget title. Verify the link goes [here](https://austin.maps.arcgis.com/apps/instant/interactivelegend/index.html?appid=ea84226e6e5a4ecf998082b73b8c6cca).

Screen Shot 2023-03-27 at 11 49 43 AM

---
#### Ship list
- [x] Code reviewed
- [x] Product manager approved

### CODE REVIEW

File: atd-vze/src/views/VZDashboard/VZDashboard.js [hunk 1]
```diff
@@ -140,6 +140,14 @@ function VZDashboard() {
             link="https://visionzero.austin.gov/viewer/"
             target="_vzv"
           />
+          <VZLinksWidget
+            header={`Access Management Crashes`}
+            mainText={`Summary of crashes for individual locations`}
+            icon="fa fa-map"
+            color="primary"
+            link="https://austin.maps.arcgis.com/apps/instant/interactivelegend/index.html?appid=ea84226e6e5a4ecf998082b73b8c6cca"
+            target="_arcgis"
```
Comment (roseeichelmann) [inline atd-vze/src/views/VZDashboard/VZDashboard.js:149]: can anyone explain what `target="_arcgis"` does? this is just out of my own curiosity lol. i tried looking up the target attribute for the `a` tag and read about the diff values [here](https://www.w3schools.com/tags/att_a_target.asp). you can specify your own value like we do here to open the link in a defined iframe but we dont use any iframes afaik

**Review by roseeichelmann — APPROVED** [review]
looks good and works as expected!

File: atd-vze/src/views/VZDashboard/VZDashboard.js (hunk 1 above)
Comment (patrickm02L) [reply atd-vze/src/views/VZDashboard/VZDashboard.js:149]: Rose, tbh I don't know what `target="_arcgis"` is. I was following the nomenclature of the other targets listed. ex: `_word`. 

I'm not sure if we have established targets or if there's one that describes the widget more clearly. If we do, I'm happy to change `_arcgis` to what it's supposed to be.

---

Screen Shot 2023-03-28 at 5 14 16 PM

**Review by frankhereford — APPROVED** [review]
This looks great, thanks for adding the new link! 🚢

**Review by chiaberry — APPROVED** [review]

**Pull request merged**